config/mapping/IncludedRoles.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+return [
```

Comment:
I think we should use `includedRoles` with lower case, since this is not a class. Wdyt?

Reply:
agreed :), i will note it to use this notation in the future for consistency, thanks! :)